jupyterlab_sql_editor/ipython_magic/trino/trino.py

```diff
@@ -178,6 +178,19 @@ def trino(self, line=None, cell=None, local_ns=None):
             args=args,
         )
 
+    def __convert_dictionary(self, value):
+        # If it's a list, go in each element as they could be of some other type, and continue processing it
+        if type(value) is list:
+            return [self.__convert_dictionary(element) for element in value]
+        # If it's a NamedRowTuple (weird Trino tuple), create a dictionary, extract the name from the NRT as the key, and process value
+        elif type(value) is trino.client.NamedRowTuple:
```

Comment:
Are you sure that `NamedRowTuple` doesn't have any sub-c;lasses that should also follow this code path? Maybe `isinstance()` would be a better choice than `type()`?